jenkins_jobs/trigger_rpmlint_from_versions_repo.groovy

```diff
@@ -0,0 +1,51 @@
+job('trigger_rpmlint_from_versions_repo') {
+  label('validation_slave_label')
+  concurrentBuild()
+  parameters {
+    stringParam('BUILDS_REPO_URL',
+		"https://github.com/${GITHUB_ORGANIZATION_NAME}/builds.git",
+		'URL of the builds repository.')
+    stringParam('BUILDS_REPO_COMMIT', 'master',
+		'Commit ID to checkout from the builds repository.')
+    stringParam('VERSIONS_REPO_URL',
+		"https://github.com/${GITHUB_ORGANIZATION_NAME}/versions.git",
+		'URL of the versions repository.')
+    stringParam('sha1', 'master',
+		'Commit ID to checkout from the versions repository.')
+  }
+  properties {
+    githubProjectUrl("https://github.com/${GITHUB_ORGANIZATION_NAME}/versions/")
+  }
+  scm {
+    git {
+      remote {
+	url('$BUILDS_REPO_URL')
+	refspec('+refs/pull/*:refs/remotes/origin/pr/*')
```

Comment:
This is not needed since this is triggered from the versions repo only.
Also, is the `heads` spec still fetched when you use this? See https://github.com/open-power-host-os/infrastructure/blob/master/jenkins_jobs/build_host_os.groovy#L31